zetaclient/chains/evm/observer/inbound.go

```diff
@@ -70,6 +71,7 @@ func (ob *Observer) WatchInbound() {
 
 // WatchInboundTracker gets a list of Inbound tracker suggestions from zeta-core at each tick and tries to check if the in-tx was confirmed.
 // If it was, it tries to broadcast the confirmation vote. If this zeta client has previously broadcast the vote, the tx would be rejected
+// TODO(revamp): move inbound tracker function to a separate file
```

Comment:
To clarifya bit, after splitting files, observer package (evm, or bitcoin) will have `observer.go`, `tickers.go`, `inbound.go`, `inbound_tracker.go`, `outbound.go`.
`outbound_tracker.go` is not necessary because `outbound.go` is the one dealing with it?

Reply:
Logically, are we simply using separate files for each Go routine? For example:

1. `observer.go` defines the Observer struct and basic methods.
2. `tickers.go` defines the ticker-based go routines.
3. `inbound.go` observes the inbouns.
4. `outbound.go` process outbound trackers.
5. `gas_price.go` watches gas price and post votes.
6. `inbound_tracker.go` watches inbound trackers and post votes.
7. `rpc_status.go` monitor the health of RPC endpoint.

Reply:
This looks good to me what do you think?
`tickers.go` can also be renamed into something like `watch_routines.go`

Also all of these ideas are subject to change depending on the implementation for the client base refactoring

Reply:
good. Let's go with these files.